UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -827,6 +827,62 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
             return string.Format(provider, format, formatArgs);
         }
 
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using <see cref="GlobalConfiguration.DefaultCulture" />.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format)
+        {
+            return ToString(format, GlobalConfiguration.DefaultCulture);
+        }
+
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using the specified format provider, or <see cref="GlobalConfiguration.DefaultCulture" /> if null.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <param name="formatProvider">Format to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            formatProvider = formatProvider ?? GlobalConfiguration.DefaultCulture;
+
+            int digits = 0;
+            string formatString = format;
+
+            if(string.IsNullOrEmpty(formatString))
+                formatString = "G";
+
+            formatString = formatString.ToUpperInvariant();
+
+            if(formatString.StartsWith("A") || formatString.StartsWith("S"))
+            {
+                if(formatString.Length > 1 && !int.TryParse(formatString.Substring(1), out digits))
+                    throw new FormatException($"The {format} format string is not supported.");
+
+                formatString = formatString.Substring(0, 1);
+            }
+
+            switch(formatString)
+            {
+                case "G":
+                    return ToString(formatProvider, 2);
+                case "A":
+                    var abbreviations = UnitAbbreviationsCache.Default.GetUnitAbbreviations<AccelerationUnit>(Unit, formatProvider);
+                    return abbreviations[digits];
```

Comment:
Rename `digits` to `number` maybe? Since we are using it as an index in the case of `"A"` here, and possibly for number of digits for the case of `"V"` below.

Reply:
👍 done!